internal/api/v1/application/stage.go

```diff
@@ -211,6 +219,9 @@ func (hc Controller) Stage(c *gin.Context) apierror.APIErrors {
 		Username:            username,
 		RegistryCAHash:      registryCertificateHash,
 		RegistryCASecret:    registryCertificateSecret,
+		DownloadScript:      stageMap.Data[helmchart.EpinioStageDownload],
+		UnpackScript:        stageMap.Data[helmchart.EpinioStageUnpack],
+		BuildScript:         stageMap.Data[helmchart.EpinioStageBuild],
```

Comment:
Can't we just mount the ConfigMap into the Job? https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/#populate-a-volume-with-data-stored-in-a-configmap

Reply:
Note that each script takes parameters. ... On the other hand, while writing this I came up with a way to pass them in via stub scripts which then simply source the mounted files. I.e. the hardwired script becomes

```
BLOB="%s"
source /stage/download
```

And the `%s` is filled via `fmt.Sprintf`.

Reply:
Done. Commit a996c22f33b7f3f07a8af6791c6468951a2acc64
(Also squashed history, and rebased to latest main)

Reply:
Can't we pass the needed parameters as argument to the script here? https://github.com/epinio/epinio/pull/1199/files#diff-1d5683322c8ac645535eab16a4aa0ef4a516edecf543777e7de0b8ca5f4f7c7fR572-R576

We don't need a second script to wrap it then. Just `/stage/download %s` . Alternatively, these needed parameters could be set as env variables on the Job container itself. So by simply invoking the script, it would work.

Reply:
Right. My thinking was still to complex. I believe using EV is best, less issue with quoting, and done only once.
Well, at least at this point I only have to update epinio, not the charts.

Reply:
Done. d9a870b37312c87ce2644b97eaffe4834b48f486